database/info.go

```diff
@@ -32,6 +32,13 @@ type EnumValue struct {
 type Table struct {
 	Name    string    // the original name of the table in the DB
 	Columns []*Column // ordered list of columns in this table
+	Indexes []*Index  // list of indexes in this table
+}
+
+// Index contains the definition of a database index.
+type Index struct {
+	DBName  string    // name of the index in the database
```

Comment:
So, the database package doesn't know anything about name conversions, it just dumps what it finds in the database.  So all the Name fields in this package *are* actually DBName, but because the DB doesn';t know about any other name, they're just called Name.  Which I know is kind of confusing, but I think that conceptually, it's nice to keep the ideas separate.